CONTRIBUTING.md

```diff
@@ -53,7 +53,9 @@ Before submitting a PR there are some things you can check which will hopefully
 
 - Ensure newly introduced symbols are documented and that updates to existing symbols are reflected in the documentation.
 
-- Add a link to the PR to the Bugzilla entry.
+- Add a link to the PR to the Bugzilla entry (if the dlang-bot failed to do it).
+
+- Confine a PR to addressing one Bugzilla entry, unless multiple entries are different aspects of the same bug.
```

Comment:
Unreleated? If not please also add a note to file a bug report if one does not exist (if this is not covered elsewhere in this document)

Reply:
good idea